src/pymatgen/io/exciting/inputs.py

```diff
@@ -197,7 +197,7 @@ def write_etree(self, celltype, cartesian=False, bandstr=False, symprec: float =
         """
         root = ET.Element("input")
         root.set(
-            "{http://www.w3.org/2001/XMLSchema-instance}noNamespaceSchemaLocation",
```

Comment:
I was in my dreams sleeping (3am) and suddenly woke up and realized this might be a broken change. I would have a closer look when I really wake up.

Reply:
I have reverted this: https://github.com/materialsproject/pymatgen/blob/46d843c07b7dcfd1ab25ab41a01cfa60f5ce123d/src/pymatgen/io/exciting/inputs.py#L198-L202

I believe replacing http would be a breaking change because the namespace from the xml file would stay the same.